feature/home/presentation/viewmodel/src/main/java/com/chocolatecake/viewmodel/people/PeopleDetailsUiEvent.kt

```diff
@@ -0,0 +1,11 @@
+package com.chocolatecake.viewmodel.people
+
+import com.chocolatecake.viewmodel.home.HomeUiEvent
+
+sealed interface PeopleDetailsUiEvent {
+    data class ClickMovieEvent(val itemId: Int) : PeopleDetailsUiEvent
+    data class ClickTvShowsEvent(val itemId: Int) : PeopleDetailsUiEvent
+
+    object BackNavigate : PeopleDetailsUiEvent
+
```

Comment:
if these functions are in the listener then there is no need for them as an events